.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "globals": {
+    "__DEV__": true
+  },
+  "env": {
+    "browser": true,
+    "mocha": true
+  },
+  "rules": {
+    "no-confusing-arrow": 0,
+    "react/jsx-quotes": 0,
+    "react/jsx-filename-extension": 0,
```

Comment:
Sure, feel free to keep it as it. I just want to let you know that jsx is the officially recommended file ext.